repos/system_upgrade/el7toel8/actors/verifydialogs/libraries/library.py

```diff
@@ -0,0 +1,17 @@
+from leapp.libraries.stdlib import api
+from leapp.models import DialogModel
+from leapp import reporting
+
+
+def check_dialogs(inhibit_if_no_userchoice=True):
+    results = list(api.consume(DialogModel))
+    for dialog in results:
+        sections = dialog.answerfile_sections.split(',')
+        dialog_resources = [reporting.RelatedResource('dialog', s) for s in sections]
+        report_data = [reporting.Title('Dialog choice required'),
+                       reporting.Severity(reporting.Severity.HIGH),
+                       reporting.Summary(
+                           'Several sections in answerfile are missing recorded user choices: {}'.format(
```

Comment:
I think the wording should be different here as we can have just one section there:

`Summary: Several sections in answerfile are missing recorded user choices: remove_pam_pkcs11_module_check.confirm`

What about: There is/are section(s) missing recorded user choices:

Reply:
Should be resolved in new patch